pulp_rpm/app/shared_utils.py

```diff
@@ -32,6 +34,24 @@ def _prepare_package(artifact, filename):
     return package
 
 
+def count_file_sha256(file_path):
+    """
+    Get sha256 of file.
+
+    Args:
+        file_path(string): path of file
+
+    Returns:
+        String: SHA256 of file
+
+    """
+    try:
+        with open(file_path, 'rb') as file_obj:
+            return sha256(file_obj.read()).hexdigest()
+    except FileNotFoundError:
```

Comment:
What is the case when we are fine not to have repomd file on the disk?
I think if it happens something is very off and we better break early. What do you think>

Reply:
This is not directly connected to repomd, as this is helper function takes file path to process I believe it is good practice to validate input.